tools/fast/environment/environment.go

```diff
@@ -53,17 +48,20 @@ type Environment interface {
 	// environments may create a Filecoin process that interacts with
 	// an already running filecoin node, and supplied the API multiaddr
 	// as options.
-	NewProcess(ctx context.Context, processType string, options map[string]string, eo EnvironmentOpts) (*Filecoin, error)
+	NewProcess(ctx context.Context, processType string, options map[string]string, eo fast.FilecoinOpts) (*fast.Filecoin, error)
 
 	// Processes returns a slice of all processes the environment knows
 	// about.
-	Processes() []*Filecoin
+	Processes() []*fast.Filecoin
 
 	// Teardown runs anything that the environment may need to do to
 	// be nice to the the execution area of this code.
 	Teardown(context.Context) error
 
 	// TeardownProcess runs anything that the environment may need to do
 	// to remove a process from the environment in a clean way.
-	TeardownProcess(context.Context, *Filecoin) error
+	TeardownProcess(context.Context, *fast.Filecoin) error
+
+	// GetFunds retrieves a fixed amount of tokens from the environment.
```

Comment:
This needs more explanation. The method only returns an error, so what does "get" mean? From where are they got? Where is the fixed amount specified? Are they guaranteed to have landed when this method returns?

Edit: having now read implementation, it would probably be better to be more explicit and call this `RequestFaucetFunds` or similar. The questions above still stand.